ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace PalindromeandReverse {
+   class Program {
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the number:");
+         int number = int.Parse (Console.ReadLine ());
+         int originalNumber = number;
+
+         int reversedNumber = 0;
+         while (number > 0) {
+            int digit=number% 10;
+            reversedNumber = reversedNumber * 10 + digit;

```

Comment:
```suggestion
            reversedNumber = reversedNumber * 10 + digit;
            reversedNumber = reversedNumber * 10 + (number % 10);
```

Reply:
local variable digit can be replaced as it is not used elsewhere

Reply:
Will commit and push the changes .Thank you